src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY = 10;
+    private static final int START_SIZE = 0;
+    private static final double GROWTH = 1.5;
+
+    private Object[] elementData;
+    private int size;
+
+    public ArrayList() {
+        elementData = new Object[CAPACITY];
+        size = START_SIZE;
+    }
```

Comment:
1) 0 is default int value, not need to explicitly set it
2) Better to use parametrized type for array in order to avoid type casting in code
```suggestion
    private static final int CAPACITY = 10;
    private static final double GROWTH = 1.5;

    private T[] elementData;
    private int size;

    public ArrayList() {
        elementData = (T[]) new Object[CAPACITY];
    }
```